src/libraries/System.Runtime.InteropServices/tests/Ancillary.Interop/PreserveSigMarshaller.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We can remove this file. This was only needed for a conceptual example. It's not up for API review.